app/config/collections2.php

```diff
@@ -214,7 +214,7 @@
                 'required' => false,
                 'default' => new stdClass,
                 'array' => false,
-                'filters' => ['json', 'range'],
+                'filters' => ['json', 'range', 'enum'],
```

Comment:
Shouldn't this value hold only `json`, why do we need `range` and `enum`?

Reply:
We needed some way to know if/when to extract data from `formatOptions` to modify documents to fit their response models. Additionally, this filter gives us a way to modify the document when saving to the DB, so we can adjust for the Structure validator.

`enum` filter is explained below - the `range` filter prepares `min` and `max` for ints and floats:
https://github.com/appwrite/appwrite/blob/53b8a1c245800b9df14c4dfa70327f93a692bcf3/app/init.php#L199-L219